cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -193,17 +202,24 @@ public static void editConfig() throws Exception {
                     break;
                 } else
                     System.out.println("\n--------Invalid Input!! Try Again--------");
+                    
             }
 
             // -----------Now we'll get preferred clone path on local file system from
             // user--------------
             while (true) {
                 System.out.println("\n--------Enter preferred Clonning path--------");
-                clone_path = reader.readLine();
-                // We'll check if the path enterd by user is a valid path or not
-                File tmp = new File(clone_path);
-                if (tmp.exists())
+                String clonePathInput = reader.readLine();
+                if(clonePathInput == null || clonePathInput.isEmpty()){
+                    System.out.println("Empty/blank input provided - reseting to existing/default setting");
+                    break;
+                }
+                // We'll check if the path enterd by user exists else try creating it
+                File tmp = new File(clonePathInput);
+                if (tmp.exists()||tmp.mkdir()){
```

Comment:
as above we should not create this dir during configuration should be when used/needed.

Reply:
The issue is how do we verify if the user is entering a valid path(otherwise `git clone` will fail) . Currently I am attempting to make the directory if not exist and if it fails I am asking the user to enter it again( ` System.out.println("\n--------Invalid Path!! Try Again--------")`) .